docs/src/overview/pricing/_index.md

```diff
@@ -8,10 +8,6 @@ aliases:
   - "/overview/pricing.html"
 ---
 
-{{< description >}}
```

Comment:
The description part needs to be kept, IIRC.  It doesn't get injected automatically.

Reply:
Woops!